ansible/roles/docker/defaults/main.yml

```diff
@@ -1,4 +1,4 @@
 ---
-docker_debian_version: "{{ docker.debian.version if docker_debian_version_defined else '17.03.2~ce-0~ubuntu-xenial' }}"
+docker_debian_version: "{{ docker.debian.version if docker_debian_version_defined else '5:18.09.1~3-0~ubuntu-bionic' }}"
```

Comment:
what does the 5: do here?

Reply:
That was just the name it was listed under in the apt repo; I don't know the reason why the format differed.

Reply:
docker version completely changed formatting? sounds right ✅

Reply:
I think the `5:` can be dropped here, it is the epoch number for the package: https://www.debian.org/doc/debian-policy/ch-controlfields.html#version

Reply:
If its dropped then 0 is assumed; seems like that would give you a different version then? Is there a benefit to removing it if the apt-cache policy lists it _with_ the epoch?

Reply:
As long as we aren't trying to re-use the version between RPM and DEB distros, then I don't see any issue with keeping the epoch